app/views/reports/standard_form_report/subset/_full_width_summary_row.html.erb

```diff
@@ -0,0 +1,27 @@
+<tr class="summary full_width_subhead">
```

Comment:
Can you be a champ and change these to `dash-style` CSS classes?

Reply:
i am a champion